### DISCUSSION TITLE
Warszawskie Dni Informatyki - zaproszenie oraz CFP

### DISCUSSION THREAD

**Discussion by eklime** [AzureCommunityPL/group#387] [Wydarzenia]
[POST SPONSOROWANY]

![image](https://user-images.githubusercontent.com/12485979/150534771-e74a8b1d-7e65-48d3-bc2b-f2a965b1cb42.png)

Do tego można zgłosić się na prelegenta: https://bit.ly/WDI22-CfP  

Nasza grupa tradycyjnie jako partner community się tam pojawi.

**Comment by sbugalski** [comment]
![image](https://user-images.githubusercontent.com/45152735/150543252-d4471ae7-19d1-4cf4-9b6f-36c2ece04551.png)

**Reply by eklime** [reply]
> 

tylko dla zdecydowanych ;P

**Comment by mgrabarz** [comment]
W sumie, czemu nie :)

**Comment by mmisztal1980** [comment]
Zamierzam się pojawić, skontaktowano się zw mną innym kanałem

Wysłane z iPhone'a

**Reply by eklime** [reply]
> Zamierzam się pojawić, skontaktowano się zw mną innym kanałem
> 

Ludzie komunikują się z Tobą kanałami?

**Reply by aryll** [reply]
W czasie drugiej wojny światowej większość ludzi chodziło nawet kanałami (przynajmniej w Łodzi) 😄
